packages/core-api/src/lib/AuthSessionManager/AuthSessionStore.ts

```diff
@@ -28,7 +28,7 @@ type Options<T> = {
   /** Storage key to use to store sessions */
   storageKey: string;
   /** Used to get the scope of the session */
-  sessionScopes: SessionScopesFunc<T>;
+  sessionScopes?: SessionScopesFunc<T>;
```

Comment:
Yeah this ends up being a bit awkward without the dedicated SAML classes, still better though but could be something to refactor if we get a couple of other non-OAuth providers in place.

Reply:
Yup. Definitely something that can be refactored. Who knows, someone might implement LDAP in the future 😄